content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -1,4 +1,3 @@
-"Abebeos","Provide Barrier-Free Power PC OSS-Development"
```

Comment:
This needs to be moved to this file instead: https://github.com/osuosl/osuosl-pelican/blob/master/content/services/csv/powerdev_former_open_source_projects.csv